prime-router/build.gradle.kts

```diff
@@ -76,6 +78,10 @@ tasks.test {
     }
 }
 
+tasks.withType<Test>().configureEach {
+    maxParallelForks = (Runtime.getRuntime().availableProcessors() / 2).takeIf { it > 0 } ?: 1
+}
```

Comment:
This makes our tests run in parallel.

Reply:
As a note, we have the ability to mark specific tests to not run in parallel.